server/config.ts

```diff
@@ -61,6 +61,20 @@ export default {
       systemClientId: get('SYSTEM_CLIENT_ID', 'clientid', requiredInProduction),
       systemClientSecret: get('SYSTEM_CLIENT_SECRET', 'clientsecret', requiredInProduction),
     },
+    hmppsAuthorizationServer: {
+      url: get('HMPPS_AUTHORIZATION_SERVER_URL', 'http://localhost:8089', requiredInProduction),
+      externalUrl: get(
+        'HMPPS_AUTHORIZATION_SERVER_EXTERNAL_URL',
+        get('HMPPS_AUTHORIZATION_SERVER_URL', 'http://localhost:8089'),
+      ),
+      timeout: {
+        response: Number(get('HMPPS_AUTHORIZATION_SERVER_RESPONSE', 10000)),
+        deadline: Number(get('HMPPS_AUTHORIZATION_SERVER_DEADLINE', 10000)),
+      },
+      agent: new AgentConfig(Number(get('HMPPS_AUTHORIZATION_SERVER_RESPONSE', 10000))),
+      apiClientId: get('API_CLIENT_ID', 'test-client', requiredInProduction),
```

Comment:
Don't need apiClientId and apiClientSecret defining here.  We'll still be getting authorization_code token from legacy auth for now